File: baekjoon/05_Array/4344.py
import sys

c = int(sys.stdin.readline())

avg = 0
cnt = 0
for i in range(c):
    data = list(map(int, sys.stdin.readline().split()))
    for idx in range(1,data[0]+1):
        avg += data[idx]
    avg /= data[0]
    for idx in range(1,data[0]+1):
        if data[idx] > avg:
            cnt += 1
    print("{:.3f}%".format(cnt/data[0]*100))
    cnt = 0
    avg = 0

File: C/2023/01/2023-01-10/4344.c
#include <stdio.h>

int n, m;

int main() {
    scanf("%d", &n);
    
    for (int i = 0; i < n; i++) {
        scanf("%d", &m);

        int stu[m];
        double avg = 0;
        double cnt = 0;
        
        for (int j = 0; j < m; j++) {
            scanf("%d", &stu[j]);
            avg += stu[j];    
        }
        avg = avg/m;

        for (int k = 0; k < m; k++) {
            if (stu[k] > avg) cnt++;
        }

        printf("%0.3lf%s\n",cnt/m*100,"%");
    }
}
